superset-frontend/cypress-base/cypress/integration/dashboard/tabs.test.js

```diff
@@ -164,67 +132,56 @@ describe('Dashboard tabs', () => {
     cy.wait('@treemapRequest');
 
     // apply filter
-    cy.get('.Select__control').first().should('be.visible');
-    cy.get('.Select__control').first().click({ force: true });
+    cy.get('.Select__control').first().should('be.visible').click();
     cy.get('.Select__control input[type=text]')
       .first()
-      .should('be.visible')
-      .type('South Asia{enter}', { force: true });
+      .focus()
+      .type('South Asia{enter}');
+    cy.get('.filter_box button:not(:disabled)').contains('Apply').click();
 
     // send new query from same tab
-    cy.wait('@treemapRequest').then(xhr => {
-      const requestFormData = xhr.request.body;
-      const requestParams = JSON.parse(requestFormData.get('form_data'));
+    cy.wait('@treemapRequest').then(({ request }) => {
+      const requestBody = parsePostForm(request.body);
+      const requestParams = JSON.parse(requestBody.form_data);
       expect(requestParams.extra_filters[0]).deep.eq({
         col: 'region',
-        op: 'IN',
-        val: ['South Asia'],
+        op: '==',
+        val: 'South Asia',
```

Comment:
I'm surprised this (and the others below) even worked before?

Reply:
I think previously Cypress will just let this `expect` check in `wait.then(...)` pass even if it fails.